turbo/src/components/report/Claims.svelte

```diff
@@ -11,6 +11,8 @@
 	let showMoreClaims = false;
 	let sortedClaims = [];
 	let ids = [];
+	// TODO(colleenm): support data sets that include both text and video claims
+	let hasVideo = csv && 'video' in csv[0];
```

Comment:
@lightningorb lmk if you have a better way to do this. I think for now it's ok to leave this as supporting only all-video or no-video datasets

Reply:
@colleenm this is an optimized (thus tricky) component, so i added some documentation with a fixup. hasVideo now runs when it's supposed to, and is true if any entry in the subtopic contains a video.